opt_einsum/paths.py

```diff
@@ -43,6 +43,18 @@ def __call__(self, inputs, output, size_dict, memory_limit=None):
 
     where ``path`` is a list of int-tuples specifiying a contraction order.
     """
+
+    def check_args_against_first_call(self, inputs, output, size_dict):
```

Comment:
Can we make this private? `_check` looks like an internal helper function rather than user facing.

Reply:
Yes sounds good.